tests/util/mod.rs

```diff
@@ -72,13 +72,13 @@ impl Drain for CaseTraceLogger {
     type Ok = ();
     type Err = slog::Never;
     fn log(&self, record: &Record, values: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
-        let tag = util::get_tag_from_thread_name().unwrap_or_else(|| "".into());
+        let tag = util::get_tag_from_thread_name().map_or_else(|| "".into(), |s| s + " ");
 
         let t = time::now();
-        let time_str = time::strftime("%y/%m/%d %h:%m:%s.%f", &t).unwrap();
```

Comment:
should be %Y?

Reply:
`%Y` will print `2018`. `%y` will print `18`. I think both are OK.

Reply:
`%Y` is better, because we have been using the format elsewhere.